### DISCUSSION TITLE
Node 18 using npm v10.5.0 or above

### DISCUSSION THREAD

**Discussion by balan-martinez-sp** [nodejs/help#4694] [Q&A] [answered]
Hello community, I just wanted to know how close is a new version of Node 18 using npm v10.5.0 or above. This is because some vulns like CVE-2023-42282 are still present on the last node 18 version. 

This issue was solved by an upgrade of IP package in this [npm/cli commit](https://github.com/npm/cli/pull/7242). I'll appreciate your comments on this.

**Comment by kl4072** [comment]
running into the same issue with Node 20

**Comment by marco-ippolito** [comment, marked as answer]
hello it will be fixed in the next release coming up in a few days.
https://github.com/nodejs/node/pull/52165
